gorush/Dockerfile

```diff
@@ -3,10 +3,10 @@
 # Stage1: build from source
 FROM quay.io/cybozu/golang:1.15-bionic AS build
```

Comment:
The upstream has moved to Go 1.16, right?
https://github.com/appleboy/gorush/commit/ae91ee7be1a292eed86cf2f752c442799a1403f3
```suggestion
FROM quay.io/cybozu/golang:1.16-focal AS build
```

Reply:
I merge patch into v1.13.0. Therefore failed to build with gorush 1.16

Reply:
I see.